hack/linter.sh

```diff
@@ -9,7 +9,7 @@ fi
 
 linterVersion="$(golangci-lint --version | awk '{print $4}')"
 
-if [[ ! "${linterVersion}" =~ ^1\.5[89] ]]; then
+if [[ ! "${linterVersion}" =~ ^v1\.5[89] ]]; then
```

Comment:
I'd say this file is deprecated. But if you want to update it you should change so it checks for version 1.61, since that's what is used by the the pipeline  https://github.com/jenkins-x/jx3-pipeline-catalog/blob/master/tasks/go/pullrequest.yaml#L44